homeassistant/components/lovelace/const.py

```diff
@@ -35,6 +39,42 @@
     vol.Optional(CONF_URL): cv.string,
 }
 
+CONF_SIDEBAR = "sidebar"
+CONF_TITLE = "title"
+CONF_REQUIRE_ADMIN = "require_admin"
+
+DASHBOARD_BASE_FIELDS = {
+    vol.Optional(CONF_REQUIRE_ADMIN, default=False): cv.boolean,
+    vol.Optional(CONF_SIDEBAR): {
```

Comment:
Shouldn't sidebar also accept `null` to update a view to not show it in the sidebar anymore?

Reply:
fixed